src/veileder/kandidatliste/kandidatrad/utfall-med-endre-ikon/UtfallMedEndreIkon.tsx

```diff
@@ -25,12 +25,13 @@ const UtfallMedEndreIkon: FunctionComponent<Props> = ({
 }) => {
     return (
         <>
+            <UtfallLabel utfall={utfall} />
```

Comment:
Hvis `kanEndreUtfall` er `false`, vil vi ikke se to `UtfallLabel` da?

Reply:
Det har du helt rett i! Skal få rettet opp i det.